mache/spack/__init__.py

```diff
@@ -195,7 +191,6 @@ def get_spack_script(spack_path, env_name, compiler, mpi, shell, machine=None,
         Whether to include the same hdf5, netcdf-c, netcdf-fortran and pnetcdf
         as used in E3SM
 
-    include_system_hdf5_netcdf : bool, optional
         Whether to include the system hdf5, netcdf-c, netcdf-fortran and
         pnetcdf (not necessarily the same as E3SM)
 
```

Comment:
Remove the rest of the docstring entry as well:
```suggestion
```

Reply:
Whoops, I didn't notice that, I'll go ahead and change this in all files. Thanks for pointing that out!